webpack.config.js

```diff
@@ -37,7 +37,7 @@ module.exports = function (_, env) {
       globalObject: 'self'
     },
     resolve: {
-      extensions: ['.ts', '.tsx', '.js', '.jsx', '.scss', '.css'],
+      extensions: ['.ts', '.tsx', '.mjs', '.js', '.scss', '.css'],
```

Comment:
Just to triple-check: You removed `.jsx` intentionally from the list?

Reply:
yup.